bin/node_exporter

```diff
@@ -0,0 +1,16 @@
+#! /usr/bin/env bash
```

Comment:
Convenience script that downloads and runs https://prometheus.io/docs/guides/node-exporter/ locally when testing.  It is not strictly necessary but for testing locally.